src/Models/Environment.php

```diff
@@ -655,23 +655,28 @@ public function importFiles($url)
      * content from previous environment (e.g. test clones dev content, live
      * clones test content.)
      *
+     * @param array $params Parameters for the environment-creation workflow
+     *      string annotation Use to overwrite the default deploy message
      * @return Workflow In-progress workflow
      */
-    public function initializeBindings()
+    public function initializeBindings(array $params = [])
     {
         if ($this->id == 'test') {
             $from_env_id = 'dev';
         } elseif ($this->id == 'live') {
             $from_env_id = 'test';
         }
 
-        $params = [
-            'annotation' => "Create the {$this->id} environment",
-            'clone_database' => ['from_environment' => $from_env_id,],
-            'clone_files' => ['from_environment' => $from_env_id,],
-        ];
+        $parameters = array_merge(
```

Comment:
Just so I'm getting my PHP right - `annotation` will be overridden in the `$params` if it exists, this first array is acting as the default?

Reply:
That's right. I had to look it up before using it, myself.